lost/src/main/java/com/mapzen/android/lost/internal/FusedLocationProviderApiImpl.java

```diff
@@ -178,23 +179,21 @@ private void requestLocationUpdatesInternal(LocationRequest request) {
     }
   }
 
-  private void removeLocationUpdatesInternal(Set<LocationRequest> requests) {
+  private void removeLocationUpdatesInternal(List<LocationRequest> requests) {
     if (requests == null) {
       return;
     }
-    for (LocationRequest request : requests) {
-      try {
-        service.removeLocationUpdates(request);
-      } catch (RemoteException e) {
-        throw new RuntimeException(e);
-      }
+    try {
+      service.removeLocationUpdates(requests);
```

Comment:
This is great should really cut down on thrashing of the `LocationManager` listeners when multiple requests are removed!